packages/tools/pie-icons-vue/package.json

```diff
@@ -39,7 +39,8 @@
     "lint:scripts": "run -T eslint .",
     "lint:scripts:fix": "yarn lint:scripts --fix",
     "prebuild:example": "yarn build:icons",
-    "prepublishOnly": "yarn lint:scripts && yarn build"
+    "prepublishOnly": "yarn lint:scripts && yarn build",
+    "test": "run -T jest"
```

Comment:
Side note: Any reason you're not using the much more modern Vitest (which has the same programmatic API as Jest)

Reply:
I think it's a good idea since it will allow us to import components from ES modules out of the box.

Reply:
Exactly :) And it's generally more modern and shows nicer output. Easier to integrate with  coverage tools as well (Core team uses C8 which is default from the docs I think).